lib/clustering/cluster_getter.rb

```diff
@@ -3,50 +3,59 @@
 require "cluster"
 
 module Clustering
-  # This class provides a Cluster that contains the OCNs you want. There are
-  # effectively three strategies:
+  # This class provides a Cluster that contains all the given OCNs. Thus,
+  # a call to ClusterGetter.get(@ocns) effectively says: these OCNs belong
+  # together in a cluster.
   #
-  # - Fetch an existing cluster that has the OCNs you want
-  # - Merge multiple clusters together to get a single one with the OCNs you want
+  # TODO: consider renaming to ClusterOCNs.cluster(@ocns)?
```

Comment:
Would like to address this TODO before merge

Reply:
See comment above: https://github.com/hathitrust/holdings-backend/pull/325#discussion_r1924181218 -- will discuss and see what we want to do.

Reply:
We will move to `Cluster.cluster_ocns!` when implementing `merge`.